tools/tertiary-analysis/scater/scater-calculate-qc-metrics.xml

```diff
@@ -1,4 +1,4 @@
-<tool id="scater-calculate-qc-metrics" name="Scater calculate qc metrics" version="1.6.0+galaxy1">
```

Comment:
Don't downgrade the galaxy1 to galaxy0. Please define @TOOL_VERSION@ in the macros or elsewhere if you want to use it.